packages/pds/src/config/env.ts

```diff
@@ -0,0 +1,173 @@
+import { envInt, envStr, envBool, envList } from './util'
+
+export const readEnv = (): ServerEnvironment => {
+  return {
+    // service
+    port: envInt('PDS_PORT'),
+    hostname: envStr('PDS_HOSTNAME'),
+    serviceDid: envStr('PDS_SERVICE_DID'),
+    version: envStr('PDS_VERSION'),
+    privacyPolicyUrl: envStr('PDS_PRIVACY_POLICY_URL'),
+    termsOfServiceUrl: envStr('PDS_TERMS_OF_SERVICE_URL'),
+
+    // db: one required
+    // sqlite
+    dbSqliteLocation: envStr('PDS_DB_SQLITE_LOCATION'),
+    // postgres
+    dbPostgresUrl: envStr('PDS_DB_POSTGRES_URL'),
+    dbPostgresMigrationUrl: envStr('PDS_DB_POSTGRES_MIGRATION_URL'),
+    dbPostgresSchema: envStr('PDS_DB_POSTGRES_SCHEMA'),
+    dbPostgresPoolSize: envInt('PDS_DB_POSTGRES_POOL_SIZE'),
+    dbPostgresPoolMaxUses: envInt('PDS_DB_POSTGRES_POOL_MAX_USES'),
+    dbPostgresPoolIdleTimeoutMs: envInt('PDS_DB_POSTGRES_POOL_IDLE_TIMEOUT_MS'),
+
+    // blobstore: one required
+    // s3
+    blobstoreS3Bucket: envStr('PDS_BLOBSTORE_S3_BUCKET'),
+    // disk
+    blobstoreDiskLocation: envStr('PDS_BLOBSTORE_DISK_LOCATION'),
+    blobstoreDiskTmpLocation: envStr('PDS_BLOBSTORE_DISK_TMP_LOCATION'),
+
+    // identity
+    didPlcUrl: envStr('PDS_DID_PLC_URL'),
```

Comment:
in all the golang code we call this `ATP_PLC_HOST`; fine if they are different or service-specific though.

Reply:
yeah we'll probably just keep it server-specific

especially since we already have the `PDS_` prefix on all our env vars